packages/esm-patient-vitals-app/src/vitals-and-biometrics-header/vitals-header.scss

```
@@ -2,26 +2,34 @@
 @use '@carbon/type';
 @import '@openmrs/esm-styleguide/src/vars';
```

Shall we update the import here as you did in the refactor?
```suggestion
@use '@openmrs/esm-styleguide/src/vars' as *;
```